note-folder

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
-required_date="${1:-today}"
-
 rootFolder="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-rootFolder="$( cat "${rootFolder}/.noteFolderPath")"
-if [ ! -z $rootFolder ]; then
-    echo "${rootFolder}"
+notesFolder="$( cat "${rootFolder}/.noteFolderPath")"
+if [ ! -z "${notesFolder}" ]; then
+    echo "${notesFolder}"
```

Comment:
I am not 100% sure what purposes does this server but I decided to keep the behavior the same, just renamed the variables, related to: https://github.com/drawm/daily-notes/pull/2/files#r419174055

Let me know what you think :D !

Reply:
The new name makes a lot of sense! Its way better the the original imo

That snippet of code is duplicated in a couple of places, can you rename all of them?